metrics/ci_integrations/test/destination/firestore/test_utils/mock/firebase_auth_mock.dart

```diff
@@ -0,0 +1,7 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:firedart/firedart.dart';
+import 'package:mockito/mockito.dart';
+
+class FirebaseAuthMock extends Mock implements FirebaseAuth {}
```

Comment:
Should we replace all `FirebaseAuthMock`s usages with the instance of this mock?